scripts/push-docker.sh

```diff
@@ -3,19 +3,22 @@
 # Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 # See LICENSE.txt for license information.
 
-set -e
-set -u
+set -eoux
 
-if [[ -z "${CIRCLE_TAG:-}" ]]; then
-  echo "Pushing lastest for $CIRCLE_BRANCH..."
-  TAG=latest
-else
-  echo "Pushing release $CIRCLE_TAG..."
-  TAG="$CIRCLE_TAG"
+if [ "$TAG" = "" ]; then
+    echo "TAG was not provided"
+    exit 1
 fi
 
-echo $DOCKER_PASSWORD | docker login --username $DOCKER_USERNAME --password-stdin
+echo $DOCKERHUB_TOKEN | docker login --username $DOCKERHUB_USERNAME --password-stdin
```

Comment:
```
echo $DOCKERHUB_TOKEN | docker login --username $DOCKERHUB_USERNAME --password-stdin
     ^--------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                                ^-----------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```

Shellcheck output